docs/CONTRIBUTING.md

```diff
@@ -153,3 +279,101 @@ Utilisation des __px__ pour le positionnement: padding, border, margin.
  padding: 10px 12px;
 }
 ```
+
+### Javascript Ember.js
+
+##### Components
+
+###### Création de classes css
+Privilégier la création de classes CSS dans le `.hbs` plutôt que dans le `.js`
+
+```javascript
+// BAD 
+export default Component.extend({
+  classNames: ['hexagon-score'], // qui va rajouter cette class à la div créée par Ember pour injecter le component
+});
+```
+
+```html
+// BAD
+<div class="hexagon-score__content">
+  <div class="hexagon-score-content__title">PIX</div>
+  <div class="hexagon-score-content__pix-score">{{score}}</div>
+  <div class="hexagon-score-content__pix-total">1024</div>
+</div>
+```
+
+```javascript
+// GOOD
+export default Component.extend({
+  // component stuff
+});
+```
+
+```html
+// GOOD
+<div class="hexagon-score">
+  <div class="hexagon-score__content">
+    <div class="hexagon-score-content__title">PIX</div>
+    <div class="hexagon-score-content__pix-score">{{score}}</div>
+    <div class="hexagon-score-content__pix-total">1024</div>
+  </div>
+</div>
+```
+
+##### Routes
+
+###### Accéder à une ressource
+
+Privilégier l'utilisation de `this.store` plutôt que `this.get('store')`
```

Comment:
Pas besoin de mettre ça ici, la mise à jour d'Ember le met dans les règles de son linter

Reply:
Ca me semble utile, si certaines personnes tombent sur de la doc un peu ancienne. Normalement le linter devrait faire, mais ca ne me gene pas tant que ce n'est pas une habitude

Reply:
Ca alourdit le contributing inutilement, le but c'est quand même d'avoir quelque chose de synthétique.

Reply:
On retrouve encore souvent des `this.get('store')`dans le code. Personnellement ça me permet de me poser la question d'un refacto si je tombe dessus.
Et plus généralement ça ne me dérange pas de remplir le contributing.md maintenant, quitte à faire le ménage quand on constatera qu'il devient trop lourd.

Reply:
Ils vont disparaître avec la mise à niveau de mon-pix normalement.

Reply:
Ils ont disparu normalement.

Reply:
surtout que le sujet est plus complexe qu'il n'y parait. En réalité, on peut utiliser `this.prop` sur tous les `EmberObject` qui ne sont pas des `Proxy` ou des `computedProperty`. En fait sur toute propriété donc l'accès déclenche autre chose qu'une simple lecture de la valeur, donc pas que `this.store`.

Reply:
En y réfléchissant, si tous tes objets sont des `EmberObject` à priori, ember sait qu'est ce qui est une computed property ou non. Donc je dirai que c'est safe de virer le get pour tout sauf pour les `Proxy` dont EmberData use et abuse : )

Reply:
A voir si on a besoin de le rajouter plus tard